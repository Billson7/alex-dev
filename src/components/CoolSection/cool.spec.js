import React from "react"
import { render } from "@testing-library/react"
import { Cool } from "./"

test("renders Cool section correctly", () => {
  const { container } = render(
    <Cool heading="heading" caption="caption" intro="intro" />
  )

  expect(container).toMatchSnapshot()
})
