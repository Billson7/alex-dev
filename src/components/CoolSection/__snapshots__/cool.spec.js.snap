// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Cool section correctly 1`] = `
<div>
  <section
    class="section section-cool"
  >
    <div
      class="flex-row"
    >
      <h2
        class="profile-heading"
      >
        Something cool
      </h2>
      <p
        class="profile-caption"
      >
        coming soon
      </p>
    </div>
  </section>
</div>
`;
